src/H5WebViewer.ts

```diff
@@ -93,7 +93,7 @@ export default class H5WebViewer
     const { extensionPath, extensionUri } = this.context;
     const { cspSource } = webview;
 
-    const manifest = require(join(extensionPath, 'dist/manifest.json'));
+    const manifest = require(join(extensionPath, 'dist/.vite/manifest.json'));
```

Comment:
The only breaking change I encountered by upgrading Vite from v2 to v6 is that the location of the `manifest.json` file has changed.